### PR TITLE
fix: show CronJob properties

### PR DESCRIPTION
**Pull request by Crystalix007**
Show the properties of CronJobs when running
`kustomize cfg tree` with the `--image` flag (or other resource flag).

### CODE REVIEW

File: cmd/config/internal/commands/tree_test.go [hunk 1]
```diff
@@ -556,3 +556,107 @@ spec:
 		return
 	}
 }
+
+// TestTreeCommand_images tests that the image specified by various workloads.
```
Comment (joehewett) [inline cmd/config/internal/commands/tree_test.go:560]: This could be reworded to be clearer

**Comment by bilaalrashid** [comment]
Unsure of the state of this, but getting this PR released would be really useful for my team as well. Any chance this can be looked at again @ncapps?

**Comment by Crystalix007** [comment]
@ncapps I believe (according to [the testing & merge workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#the-testing-and-merge-workflow)) that you might need to run `/retest` instead.

Also, do you know what the next step would be to progress this?

**Comment by ncapps** [comment]
Hi @koba1t, would you or another approver please review this when you get a chance?

**Comment by koba1t** [comment]
Hi @Crystalix007 
Thanks for your contribution!

/approve

**Pull request merged**